src/main/java/org/jenkinsci/maven/plugins/hpi/HplMojo.java

```diff
@@ -142,6 +142,10 @@ private void buildLibraries(List<String> paths) throws IOException {
                     continue OUTER;
             }
 
+            //Skip artifacts of type pom
+            if(artifact.getType().equals("pom"))
```

Comment:
In general IMO when comparing with a static operand it's always preferable to use the inverse form `"pom".equals(artifact.getType())` so that there's no chance of NPE (well, here if artifact is never null).